textile/features.textile

```diff
@@ -675,7 +675,8 @@ h3(#realtime-channel). RealtimeChannel
 * @(RTL7)@ @RealtimeChannel#subscribe@ function:
 ** @(RTL7a)@ Subscribe with a single listener argument subscribes a listener to all messages
 ** @(RTL7b)@ Subscribe with a name argument and a listener argument subscribes a listener to only messages whose @name@ member matches the string name
-** @(RTL7c)@ Implicitly attaches the @RealtimeChannel@ if the channel is in the @INITIALIZED@ state. The optional callback, if provided, is called according to "@RTL4d@":#RTL4d based on the implicit attach operation. The listener will always be registered regardless of the implicit attach result
+** @(RTL7c)@ This clause has been deleted. It was valid up to and including specification version @3.0@.
```

Comment:
I would treat this change as clarification, since we implemented this spec point in ably-js differently, and do not treat this as spec update

Reply:
https://github.com/ably/specification/pull/192/commits/454a765188cf8c12a0f7939ec2c383f9cd130181